README.md

```diff
@@ -242,18 +242,85 @@ Also missing are the following.
+* Assign a static IP address to your Raspberry-Pi system by adding the following lines in the rootfs/etc/dhcpcd.conf file (found in you microSD card) according to your router settings:
+```bash
+interface wlan0
+static ip_address=192.168.0.[2-254]/24
+static routers=192.168.0.1
+static domain_name_servers=192.168.0.1
+```
+* Copy the wpa_supplicant.conf from the rootfs/etc/wpa_supplicant/wpa_supplicant.conf to the boot/ directory of the new image to unblock to WiFi:
+```bash
+sudo cp /media/username/rootfs/etc/wpa_supplicant/wpa_supplicant.conf /media/username/boot/
+```
+* Add the microSD card and boot your Raspberry-Pi device
+
+Then execute the following steps from your personal computer towards
+the Raspberry-Pi with the image created above:
+
+* Install the following packages on your personal computer:
+```bash
+sudo apt-get install git ansible
+```
+* In the /etc/ansible/ansible.cfg file of you personal computer uncomment and set as True the following:
+```bash
+allow_world_readable_tmpfiles = True
+pipelining = True
+```
+* In the /etc/ansible/hosts file of your personal computer create the group ([epidose]) and add the  usernames and IP addresses (for more information check this [link](https://docs.ansible.com/ansible/2.3/intro_inventory.html)) for each Raspberry-Pi device as follows:
+```bash
+   [epidose]
+   192.168.0.105 ansible_user=pi ansible_connection=ssh ansible_private_key_file=~/ssh/id_rsa
+```
+The ansible_private_key_file is the path to the private key of your preference.
+Alternatively, you may [create ](https://www.ssh.com/ssh/keygen/) a new private/public key pair.
+
+* Execute the following commands (see explanation below about the ansible command-line arguments):
+```bash
+git clone https://github.com/eellak/epidose
+cd epidose/epidose/device
+sudo ansible-playbook install_and_configure.yml --tags "setup" --extra-vars "public_key_path=path_to_the_public_key"
+sudo ansible-playbook install_and_configure.yml --tags "production|development" --extra-vars "public_key_path=path_to_the_public_key epidose_network_psk=pre-shared_wifi_key_of_epidose_network eduroam_network_psk=pre-shared_wifi_key_of_eduroam_network cuckoo_distribution_server_psk=pre-shared_wifi_key_of_cuckoo_distribution_network"
+```
+_tags_
+* setup enables a passwordless ssh for all the node usernames and IP addresses found inside the /etc/ansible/hosts, we recommend to exeute this tag first without any extra-vars
+* production configures and installs the necessary libraries to run epidose, all the extra-vars are mandatory to execute this tag
+* development configures and install all libraries needed to develop the epidose, all the extra-vars are mandatory to execute this tag
+
+_extra-vars_
+* public_key_path is path where the public key is found and it should be the key of the private key added in the /etc/ansible/hosts 
+* epidose_network_psk is the WiFi pre-shared key of the epidose network
```

Comment:
What is the epidose network?
What do you mean by pre-shared?
Consider adding a separate section with prerequisites for a given installation where these are explained.

Reply:
1) Epidose network is the that we had on RPi when I received it. Does this suppose to be the user's  WiFi SSID?
2) Konstantinos informed me that the PSK was not clear for him and on the Internet I found that PSK stands for the Pre-Shared Key. Maybe it is better to change the extra-vars, let's say, to _eduroam_network_password_?
3) What do you mean a given installation? Like with specific tags?

Reply:
1. Ah, OK.  Remove this entirely.  We have WPS.
2. Yes.
3. Yes installing for production and for development, right?